### PR TITLE
Persist sessions

### PR DESCRIPTION
**Pull request by sechmann**

### CODE REVIEW

File: apiserver/api/api_test.go [hunk 1]
```diff
@@ -105,6 +107,49 @@ func TestUpdateDeviceHealth(t *testing.T) {
...
+	if err := db.AddDevice(ctx, device); err != nil {
+		t.Fatalf("Adding device: %v", err)
+	}
+
+	device.Healthy = boolp(true)
+
+	err := db.UpdateDeviceStatus([]database.Device{device})
+	assert.NoError(t, err)
+
+	// Read form db as we need the device ID
```
Comment (jhrv) [inline apiserver/api/api_test.go]: from

File: apiserver/api/api_test.go [hunk 2]
```diff
@@ -105,6 +107,49 @@ func TestUpdateDeviceHealth(t *testing.T) {
...
+		Expiry: time.Now().Add(time.Minute).Unix(),
+		Device: databaseDevice,
+		Groups: []string{"group1", "group2"},
+	}
+	if err := db.AddSessionInfo(ctx, &databaseSessionInfo); err != nil {
+		t.Fatalf("Adding SessionInfo: %v", err)
+	}
+
+	gateways := getDeviceConfig(t, router, "dbSessionKey")
+	fmt.Printf("gateways: %v", gateways)
```
Comment (jhrv) [inline apiserver/api/api_test.go]: remove

**Review by jhrv — APPROVED** [review]
lgtm

**Comment by sechmann** [comment]
Manually merged by cherry picking 650a58c

**Pull request closed without merging**